remote-config/src/main/kotlin/com/rakuten/tech/mobile/remoteconfig/RealRemoteConfig.kt

```diff
@@ -1,2 +1,5 @@
 package com.rakuten.tech.mobile.remoteconfig
 
+class RealRemoteConfig {
```

Comment:
why prefix with `Real` here?

Reply:
I think this is the new standard we're using for Android SDKs instead of `RemoteConfigImpl`. It's following the example of OkHttp: https://github.com/square/okhttp/blob/master/okhttp/src/main/java/okhttp3/RealCall.kt